src/containers/EventPopUp/EventPopUp.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 import { getEvent } from '../../thunks/getEvent';
 import { changeUserEvent } from '../../thunks/changeUserEvent';
 import { getUserEvents } from '../../thunks/getUserEvents';
+import ReactTooltip from 'react-tooltip';
```

Comment:
So excited to have this functionality! It looks great and will be so useful for our users.